Sprint-3/revise/implement/password-validator.test.js

```diff
@@ -14,3 +17,58 @@ To be valid, a password must:
 
 You must breakdown this problem in order to solve it. Find one test case first and get that working
 */
+function passwordValidation(pass) {
+  let bannedPasswords = ["ydt635RERE!!", "hdud738#&sg", "bxv735#$ls0HG"];
```

Comment:
Should avoid using invalid password in this array. When we test the function on a value, we want to be sure the function returns false because the password is a banned password, and not because the password lacks an uppercase letter.

Reply:
That's right!
I changed "hdud738#&sg" to "hdud738#&sG". and ran the test. it returns "Invalid password" again,  not because of the lack of uppercase, but because of it is banned!
Thanks CJ.